qctrlopencontrols/driven_controls/driven_control.py

```diff
@@ -330,6 +333,72 @@ def duration(self) -> float:
 
         return np.sum(self.durations)
 
+    def sample(
+        self,
+        sample_times: Union[np.ndarray, float],
+        coordinates: str = Coordinate.CYLINDRICAL.value,
+    ) -> Tuple[np.ndarray, np.ndarray, np.ndarray]:
+        r"""
+        Returns samples from the control.
+
+        Parameters
+        ----------
+        sample_times : np.ndarray or float
+            The times at which to sample, :math:`\{t_n\}`. You can pass either an array of times or
+            a single number :math:`dt`. In the latter case, the sample times are taken as
+            :math:`0, dt, 2dt, \dots`.
+        coordinates : str, optional
+            The coordinate system in which to return the sampled control. Must be 'cylindrical' or
+            'cartesian'. Defaults to 'cylindrical'.
+
+        Returns
+        -------
+        np.ndarray
+            For cylindrical coordinates, the sampled Rabi rates :math:`\{\Omega(t_n)\}`. For
+            Cartesian coordinates, the sampled x-amplitudes, :math:`\{\Omega(t_n) \cos \phi(t_n)\}`.
+        np.ndarray
+            For cylindrical coordinates, the sampled azimuthal angles :math:\{\phi(t_n)\}`. For
+            Cartesian coordinates, The sampled y-amplitudes, :math:`\{\Omega(t_n) \sin \phi(t_n)\}`.
+        np.ndarray
+            The sampled detunings, :math:`\{\Delta(t_n)\}`.
```

Comment:
We can name the return variables when there are more than one of them. https://numpydoc.readthedocs.io/en/latest/format.html#returns
```suggestion
        -------
        amplitude_x or rabi_rates : np.ndarray
            For cylindrical coordinates, the sampled Rabi rates :math:`\{\Omega(t_n)\}`. For
            Cartesian coordinates, the sampled x-amplitudes, :math:`\{\Omega(t_n) \cos \phi(t_n)\}`.
        amplitude_y or azimuthal_angles : np.ndarray
            For cylindrical coordinates, the sampled azimuthal angles :math:\{\phi(t_n)\}`. For
            Cartesian coordinates, The sampled y-amplitudes, :math:`\{\Omega(t_n) \sin \phi(t_n)\}`.
        detunings : np.ndarray
            The sampled detunings, :math:`\{\Delta(t_n)\}`.
```

Reply:
Thanks!